src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -10012,6 +10016,9 @@ namespace OutputReportTabular {
 				if ( Zone( iZone ).FloorArea > 0 && usezoneFloorArea ) {
 					tableBody( 12, iZone ) = RealToStr( totPlugProcess * Wm2_unitConv / Zone( iZone ).FloorArea, 4 );
 				}
+				// Hybrid Model Multiplier Added by Sang Hoon Lee August 2015
+				HMMultiplier = Zone( iZone ).ZoneVolCapMultpSensHMAverage;
```

Comment:
@JasonGlazer @macumber Are we OK with table reports that have optional columns?  In this, case, two extra columns would be added to the Input Verification and Results Summary, Zone Summary table when the hybrid zone model is used.  I can't remember if we have any table reports with a variable number of columns.